fwupdate.py

```diff
@@ -46,20 +46,26 @@ def parse_args() -> argparse.Namespace:
         help="Optional argument of type string for device. Default is 'eno4'.",
     )
     parser.add_argument(
+        "-P",
         "--prompt",
         action="store_true",
         help="If set, start DHCP/TFTP/HTTP services and wait for the user to press ENTER. This can be used to manually setup the host side serving the firmware.",
     )
     parser.add_argument(
         "-B",
         "--boot-device",
-        choices=["primary", "secondary"],
+        choices=["1", "2", "primary", "secondary"],
         default="secondary",
         help='Select primary or secondary boot device. Defaults to "secondary".',
     )
 
     args = parser.parse_args()
 
+    if args.boot_device == "1":
+        args.boot_device = "primary"
+    else:
```

Comment:
Should be `else if` otherwise when boot_device = primary it would become secondary.

Reply:
of course. fixed.